Tests/test_api/base.py

```diff
@@ -48,6 +49,6 @@ def create_app(self):
 
         self.app = Flask(__name__)
         self.app.config['TESTING'] = True
-        self.flask_api = FlaskAPI(self.app, port=5000, brain=brain)
+        self.flask_api = FlaskAPI(self.app, listen=self.settings.listen port=self.settings.port, brain=brain)
```

Comment:
Missing comma. Have you tested this code? How can it work without a comma here between listen and port?

Reply:
Hmm, yes - but I did...but I did quite a bit of juggling with different changes (hint: upcoming PRs), although I didn't intentionally edit my diff/patch files this was probably the cause (it was quite late at some points). I will push a correcting commit.

Reply:
Ah...I found the "problem": I applied an old version of the diff/patch file I had. At that time I haven't had adapted the unit tests yet (and obviously not run them). I'll merge to my current version...sorry and thank you for the heads up.